build.sbt

```diff
@@ -236,6 +273,42 @@ lazy val springBootStarter = project
       "-noqualifier",
       "java.lang"))
 
+lazy val springBootDevToolsStarter = project
+  .in(file("sdk/spring-boot-devtools-starter"))
```

Comment:
This is another started that adds some spring-boot logic to start the container when we the Spring application starts with `mvn spring-boot:run`.

I tried to shade this one and then exclude when packing, but run into too many issues.  
Shading doesn't play well with some Spring classes. In some cases, sbt-assembly cannot change the package of some spring framework classes. I could solve it after some hacks and juggling. But solution was too fragile, IMO.

Also, excluding it from packaging didn't work at all. The `spring-boot-plugin` will exclude it when running the app. Not useful for us.